lib/gears/object.lua

```diff
@@ -21,10 +21,10 @@ local function check(obj)
 end
 
 --- Find a given signal
```

Comment:
The commit message lies a bit. `@param` works just fine in ldoc as well. How about "Improve documentation"?